packages/core/src/tools/create-logger.js

```diff
@@ -119,10 +123,18 @@ const buildSensitiveValues = (event, data) => {
     }
     return true;
   });
+
+  // For log extra data, we only want to censor with sensitive keys. URLs in log
+  // extra should be safe.
+  const sensitiveLogData = recurseExtract(
+    data,
+    (key, value) => _.isString(value) && isSensitiveKey(key)
```

Comment:
sensitive key -> numeric value should also be caught. eg `{secretNumber: 123456789}`